src/native/minipal/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+include(configure.cmake)
+
+set(SOURCES
+    cpufeatures.c
+    random.c
+    debugger.c
+    time.c
+    unicodedata.c
+    utf8.c
+)
+
+# Provide an object library for scenarios where we ship static libraries
+include_directories(${CLR_SRC_NATIVE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+
+add_library(minipal_objects OBJECT ${SOURCES})
+set_target_properties(minipal_objects PROPERTIES CLR_CONTROL_FLOW_GUARD ON)
```

Comment:
Why do we need to specify this argument? I thought we were consuming the default configurations from `eng/`?

Reply:
I was thinking about how the nativeaot's tree changes the default (NativeAOT defaults to `OFF`). We don't need to be explicit here because this is always included from the CMake root (where we have the right defaults)